### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by sidshekhar**
- [x] Added new filters documentation
- [x] Added list of common entities to documentation
- [x] Added new 'or' predicate for Entity filtering
- [x] Fixed 'or' predicate for Label filtering

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -25,4 +27,17 @@ ta.streams.transactionsWithLabelsAndPrice.subscribe(console.log,
...
+Just add the CDN link to the head of your HTML document. In the below example, the function console.log is called everytime a new event arrives on that stream.
+
+```
+<head>
+  <script src="https://s3.amazonaws.com/cdn.tokenanalyst.io/sdk.latest.js"></script>
+  <script>
+    const t = new ta.TokenAnalyst();
+    t.streams.transactionsWithLabelsAndPrice.subscribe(console.log);
+  </script>
+</head>
```
Comment (ankitchiplunkar) [inline docs/README.md]: can we also add the link to codepen here?

File: lib/predicates.ts [hunk 2]
```diff
@@ -93,7 +93,7 @@ export class FromLabelExists extends Predicate {
 
 export class FromOrToLabelExists extends Predicate {
   isTrue(data: any): boolean {
-    return data.FROMLABEL != null || data.FROMLABEL != null;
+    return data.FROMLABEL != null || data.TOLABEL != null;
```
Comment (jpzk) [inline lib/predicates.ts]: nice!

File: lib/predicates.ts [hunk 3]
```diff
@@ -135,6 +135,12 @@ export class ToEntityExists extends Predicate {
   } 
 }
 
+export class FromOrToEntityExists extends Predicate {
+  isTrue(data: any): boolean {
+    return (data.FROMENTITY != "" && data.FROMENTITY != null) || (data.TOENTITY != "" && data.TOENTITY != null);
```
Comment (jpzk) [inline lib/predicates.ts]: sweet!

**Comment by jpzk** [comment]
LGTM

File: docs/README.md (hunk 1 above)
Comment (sidshekhar) [reply docs/README.md]: yep - as soon as we get the https issue sorted - that's the plan

**Pull request merged**